mne/bem.py

```diff
@@ -332,3 +337,108 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
```

Comment:
Here if the function `make_watershed_bem` is not given a `subject` argument, it checks for the presence of the `subject` key in the environment or in the mne configuration, and if not it raises an error through the function `get_config`.
Instead if the function `make_watershed_bem` is used through the script `mne_watershed_bem.py ` then it is prevented from running unless is explicilty given the `--subject [subject]` option.
Is it fine like that? Or should I modify the behaviour of the function?